examples/centralized_design/main.tf

```diff
@@ -288,6 +228,22 @@ module "public_nlb" {
   tags = var.tags
 }
 
+### IAM ###
+
+module "iam" {
+  source = "../../modules/iam"
+
+  for_each = var.iam_policies
+
+  name_prefix             = var.name_prefix
+  tags                    = var.tags
+  role_name               = each.value.role_name
+  create_instance_profile = try(each.value.create_instance_profile, false)
+  instance_profile_name   = try(each.value.instance_profile_name, null)
+  policy_arn              = try(each.value.policy_arn, null)
+  create_vmseries_policy  = try(each.value.create_vmseries_policy, false)
+}
+
```

Comment:
Can we have all options that IAM module supports here in the way we do not need to touch main.tf while doing custom deployment. For example module supports "custom_role". Let's have it here and do not use if it is not set by the user. The same applies to all examples